molstar/example-gallery
# List entities example

## Overview
This PR creates a new example for inspecting a new structure and listing many of its entities and their properties.

## Changes made
- Created inspect example "list_entities"
  - Count the number of waters
  - Retrieve the names and residue codes for all **covalent** ligands
  - Count the number of AltLoc positions
  - Retrieve the names and chains of all polymer entities

## How to test
Ensure the names and counts of all the properties are correct for multiple PDBs
- By default, uses 1PTH (which has many of these)
- Can edit the PDB in the code

## Specific review requests
For each of the examples:
- Are there enough comments to sufficiently describe the process?
- Are the comments accurately describing the process?
- Are there any anti-patterns in the examples?
- Can the examples be more efficient?
